README.md

```diff
@@ -56,6 +56,80 @@ Since this library is used by Home Assistant, see <https://www.home-assistant.io
 
 So follow that advice and try to scrape all fields at once, similar to the `get_form_action_url_and_hidden_inputs` in helpers.py.
 
+## Example Usage - CLI
+
+The opower library comes with a CLI interface to get familiar with the library and read actual data from your utility.
+
+```sh
+# Show information on how to call the library CLI interface
+python -m opower --help
```

Comment:
This is already included below and should be enough for most. But if you think it's valuable having this section maybe just have this line here? Ah you should also include `pip install opower`